tests/test_routes.py

```diff
@@ -0,0 +1,22 @@
+import sys
+import requests
+
+LOCALHOST = "http://localhost:{}/"
+SERVICE_PORTS = {"INTERFACE":    {"PORT": 8000, "SITES": ["api"]},
+                 "VUE":          {"PORT": 8080, "SITES": []},
+                 "PREDICTION":   {"PORT": 8001, "SITES": ["classify/segment"]},
+                 "DOCUMENTATION":{"PORT": 8002, "SITES": []}}
+
+def check_page(url):
+    response = requests.get(url)
+    if response.status_code != 200:
```

Comment:
The canonical way of doing this with ``requests`` is using/catching ``raise_for_status`` - I think it gives you a nicer error message, but I'm not sure. Can you have a look?